compiler/test/grainfmt/comments.expected.gr

```diff
@@ -1,3 +1,9 @@
+// Comment before first attr
+/* Block comment before attr */@noPervasives // Line comment on attr
+// Comment after first attr
+@runtimeMode
+// Comment after second attr
+
```

Comment:
This is the behavior for attributes on other items as well, maybe can address in a followup?

Reply:
Please open an issue

Reply:
Wait, you've added the formatter changes. You can just change the line ending. Right? We still need a follow up to fix the other ones though.

Reply:
Looked into this and I think it would require making some invasive changes to the `print_comment_range` function to disallow double-breaks. I think we can just defer to https://github.com/grain-lang/grain/issues/2055